lms/static/js/groups/views/cohort_discussions.js

```diff
@@ -1,99 +1,98 @@
-var edx = edx || {};
-
-(function ($, _, Backbone, gettext, interpolate_text, NotificationModel, NotificationView) {
+;(function (define) {
     'use strict';
+    define(['jquery', 'underscore', 'backbone', 'gettext', 'string_utils', 'js/models/notification', 'js/views/notification'], 
```

Comment:
string_utils does not use RequireJS and puts interpolate_text into the global namespace. Therefore you can put it at the end and don't need to tie it to a variable.

Also make sure it is only included in files that are actually using interpolate_text or interpolate_ntext. cohort_editor.js and cohorts.js are the only 2 files that need the dependency.